acceptance/lib/helper.rb

```diff
@@ -193,6 +193,7 @@ def puppet_apply_as_puppet_user
     # installed.
     manifest_path = master.tmpfile("puppetserver_manifest.pp")
     herp_path = master.tmpfile("herp")
+    bin_dir = "/opt/puppetlabs/agent/bin/"
```

Comment:
It seems like `aio` is the style of agent we are running on this branch of puppet-server based on the fact that you have to implement these hacks to get it working...

I could be off-base though, @haus and @justinstoller probably know more than I on this subject. It seems to me that setting `type => aio` will probably be necessary before all is said and done.